db/schema.rb

```diff
@@ -19,7 +19,7 @@
     t.boolean  "doing_packet"
     t.boolean  "on_packet"
     t.string   "announcement"
-    t.date     "start_date",         default: '2016-08-21'
+    t.date     "start_date",         default: '2017-08-27'
```

Comment:
This seems odd to me that it was ever like this. It would be better to just set the default date in the model when creating the record.

I'm okay with this for now but ideally we would fix this for future us to not have to always change this.